api/client/version.go

```diff
@@ -49,7 +54,23 @@ func (cli *DockerCli) CmdVersion(args ...string) (err error) {
 			Status: "Template parsing error: " + err.Error()}
 	}
 
-	vd, err := cli.client.SystemVersion()
+	vd := types.VersionResponse{
```

Comment:
Now that `VersionResponse` isn't part of the interface and is just internal to the client, do you think it would make sense to move this type to `api/client/version.go` ?